blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.models import entity_features
+from mindmeld.markup import process_markup
+from mindmeld.core import FormEntity
+
+app = Application(__name__)
+
+__all__ = ['app']
+
+
+sample_user = {
+    'savings' : 500,
+    'checking' : 100,
+    'credit' : 60,
+    'autoPay' : False,
+}
+
+
+@app.handle(intent='greet')
+def greet(request, responder):
+    replies = ['Hello there! Welcome to MindMeld Bank', 'Hello, thanks for choosing MindMeld Bank.']
```

Comment:
It would be good to briefly introduce what the app can do here and give an example query or two to get the user started in the right direction. Something like `Hi there! How can I help with your banking tasks? You can try transferring money between accounts or paying a bill.` It's nice to create a list of example queries and randomly select one, so that when the user starts over they see a variety of examples.